onc-healthit/inferno-program
# Update fhir client in pg gemfile

Updates the Gemfile.postgres and Gemfile.postgres.lock, which should have been done in #342.

To test in the most basic way, run:
```
docker-compose up --build
```
and Inferno should run.

To test in a more thorough manner, undo the change introduced in https://github.com/onc-healthit/inferno-program/pull/352, then launch with ``docker-compose up --build``, then run the single patient api tests.  You should see the following for medication request:

![Screen Shot 2021-09-10 at 10 39 02 AM](https://user-images.githubusercontent.com/412901/132871215-4315ea53-6799-4ec3-9e7c-115581fe7a95.png)

Then, when you change that line of code back (undo your change), that test should now pass.

I do not think we need to update the changelog with this update, since this should have just been included in #342
